<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>256</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="postEdit">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>79</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="postButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="replyEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>79</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="replyButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>530</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
